test/jasmine/tests/calcdata_test.js

```diff
@@ -30,7 +30,7 @@ describe('calculated data and points', function() {
         });
     });
 
-    xdescribe('category ordering', function() {
+    describe('category ordering', function() {
```

Comment:
quick note: you can run [`npm run test-syntax`](https://github.com/plotly/plotly.js/blob/master/test/syntax_test.js) to check that you haven't forgotten any `fdescribe` or `xit` in your jasmine test suites.